Presentation.Web/app/components/org/user/user-delete-components/org-user-roles-table.view.html

```diff
@@ -11,7 +11,7 @@
     </tr>
     </thead>
     <tbody>
-    <tr ng-repeat="right in ctrl.model.rights | orderBy: 'right.roleName'">
+    <tr ng-repeat="right in ctrl.model.rights | orderBy: 'right.businessObjectName'">
```

Comment:
not sure this is the right approach ... especiallyif orderBy does not use the right locale. That was why the controller was ordering them in the first place. I guess the ordering in the template should be changed to use the danish locale and then remove the ordering in the controller, OR remove the ordering in the template which probably makes the controller ordering work. The orderBy directive might be kitos code (orderBy directive)